test/experimental/test_vocab.py

```diff
@@ -118,98 +130,114 @@ def test_vocab_jit(self):
     # exceptions from C++ using pybind11
-    @unittest.skipIf(platform.system() == "Windows", "Test is known to fail on Windows.")
+    @unittest.skipIf(
+        platform.system() == "Windows", "Test is known to fail on Windows."
+    )
     def test_errors_vocab_cpp(self):
-        token_to_freq = {'hello': 4, 'world': 3, 'ᑌᑎIᑕOᗪᕮ_Tᕮ᙭T': 5, 'freq_too_low': 2}
-        sorted_by_freq_tuples = sorted(token_to_freq.items(), key=lambda x: x[1], reverse=True)
+        token_to_freq = {"hello": 4, "world": 3, "ᑌᑎIᑕOᗪᕮ_Tᕮ᙭T": 5, "freq_too_low": 2}
+        sorted_by_freq_tuples = sorted(
+            token_to_freq.items(), key=lambda x: x[1], reverse=True
+        )
         c = OrderedDict(sorted_by_freq_tuples)
 
         with self.assertRaises(RuntimeError):
             # Test proper error raised when setting a token out of bounds
             v = vocab(c, min_freq=3)
-            v.insert_token('new_token', 100)
+            v.insert_token("new_token", 100)
 
         with self.assertRaises(RuntimeError):
             # Test proper error raised when looking up a token out of bounds
             v = vocab(c)
             v.lookup_token(100)
 
     def test_errors_vocab_python(self):
-        token_to_freq = {'hello': 4, 'world': 3, 'ᑌᑎIᑕOᗪᕮ_Tᕮ᙭T': 5, 'freq_too_low': 2}
-        sorted_by_freq_tuples = sorted(token_to_freq.items(), key=lambda x: x[1], reverse=True)
+        token_to_freq = {"hello": 4, "world": 3, "ᑌᑎIᑕOᗪᕮ_Tᕮ᙭T": 5, "freq_too_low": 2}
+        sorted_by_freq_tuples = sorted(
+            token_to_freq.items(), key=lambda x: x[1], reverse=True
+        )
         c = OrderedDict(sorted_by_freq_tuples)
 
         with self.assertRaises(ValueError):
             # Test proper error raised when setting unk token to None
             vocab(c, unk_token=None)
 
     def test_vocab_load_and_save(self):
-        token_to_freq = {'hello': 4, 'world': 3, 'ᑌᑎIᑕOᗪᕮ_Tᕮ᙭T': 5, 'freq_too_low': 2}
-        sorted_by_freq_tuples = sorted(token_to_freq.items(), key=lambda x: x[1], reverse=True)
+        token_to_freq = {"hello": 4, "world": 3, "ᑌᑎIᑕOᗪᕮ_Tᕮ᙭T": 5, "freq_too_low": 2}
+        sorted_by_freq_tuples = sorted(
+            token_to_freq.items(), key=lambda x: x[1], reverse=True
+        )
 
         c = OrderedDict(sorted_by_freq_tuples)
         v = vocab(c, min_freq=3)
 
-        expected_itos = ['<unk>', 'ᑌᑎIᑕOᗪᕮ_Tᕮ᙭T', 'hello', 'world']
+        expected_itos = ["<unk>", "ᑌᑎIᑕOᗪᕮ_Tᕮ᙭T", "hello", "world"]
         expected_stoi = {x: index for index, x in enumerate(expected_itos)}
 
         self.assertEqual(v.get_itos(), expected_itos)
         self.assertEqual(dict(v.get_stoi()), expected_stoi)
 
-        with self.subTest('pybind'):
-            vocab_path = os.path.join(self.test_dir, 'vocab_pybind.pt')
+        with self.subTest("pybind"):
+            vocab_path = os.path.join(self.test_dir, "vocab_pybind.pt")
             torch.save(v, vocab_path)
```

Comment:
We have the tests to cover serialization and deserialization for pybind and torchbind vocab, right?

Reply:
Ohh great catch! completely missed the presence of this test. Thanks!